makefiles/test_requests.mk

```diff
@@ -45,53 +45,22 @@ check_path_up_without_envoy: ## Checks if standalone PATH (without GUARD) is run
 # - The 'anvil' service must be configured in the '.config.yaml' file.
 
 # The following are the various ways to make requests to PATH with Envoy running:
-# - **Auth**: static API key or no auth (JWT requires a non-expired JWT token, which cannot be statically set)
-# - **Service ID**: passed as the subdomain or in the 'target-service-id' header
-# - **Endpoint ID**: passed in the URL path or in the 'endpoint-id' header
+# - **Auth**: static API key, passed in the 'Authorization' header
```

Comment:
OPTIONAL: Wdyt of splitting this file (which is growing large) into `test_requests_with_guard` and `test_requests_no_guard` to easier deliniate the two?